scripts/validation/validate.js

```diff
@@ -165,3 +176,15 @@ function assertValid(schema, subject) {
     throw new Error(errorMessage);
   }
 }
+
+async function getModifiedServices() {
+  const git = simpleGit(path.resolve(__dirname, '../..'), { maxConcurrentProcesses: 1 });
+  const modifiedFilesPathString = await git.diff([ '--name-only', 'master...', 'services/*.json' ]);
```

Comment:
Continued from [previous comment](https://github.com/ambanum/CGUs/pull/110#discussion_r479269055).

Which tests do you run? 🙂 Here are mine:

```
$ git mv services/App\ Store.json services/App_Store.json
$ npm run validate:modified
No services modified
# Apply suggested changes (remove the `...` in tree selection)
$ npm run validate:modified
 Services validation
    App_Store
…
```

```suggestion
  const modifiedFilesPathString = await git.diff([ '--name-only', 'master', 'services/*.json' ]);
```